hoomd/pytest/test_logging.py

```diff
@@ -275,10 +279,9 @@ def test_add(self, blank_logger, logged_obj, base_namespace):
 
```

Comment:
Shouldn't there be a test or set of tests somewhere making sure that when `only_default=False` is passed to the logger constructor, the behavior of the logger is correct?

In general, I don't see any tests for logger behavior when there are arguments passed to the constructor.

Reply:
That is a good point.